web/template/layout/new-client-added-to-deputyship.gotmpl

```diff
@@ -0,0 +1,26 @@
+{{ define "new-client-added-to-deputyship" }}
```

Comment:
If we are going to have a separate template for each timeline event, it would make sense to put them in their own directory, otherwise the layout directory is going to become unwieldy pretty quick